projects/example-app/src/app/auth/actions/login-page.actions.ts

```diff
@@ -0,0 +1,14 @@
+import { Action } from '@ngrx/store';
+import { User, Authenticate } from '../models/user';
+
+export enum LoginPageActionTypes {
+  Login = '[Login Page] Login',
+}
+
+export class Login implements Action {
+  readonly type = LoginPageActionTypes.Login;
+
+  constructor(readonly payload: { credentials: Authenticate }) {}

```

Comment:
Change to `public` instead of `readonly`

Reply:
Just out of curiosity why not `readonly`? 
I started doing thing after @alex-okrushko pointed this out to me.